Makefile

```diff
@@ -4,10 +4,17 @@ CLOUD9_DIR:=/var/lib/cloud9
 JEKYLL:=$(shell which jekyll)
 HTMLPROOFER:=$(shell which htmlproofer)
 NPM:=$(shell which npm)
+MRAA:=$(shell which mraa-gpio)
```

Comment:
I believe this will only make on-target. Why is it part of the documentation?

Reply:
Oh, I forget that examples are part of the repo too... These need to be split into two repos. :-(

Examples should be built when run.